sources/packages/web/src/components/generic/BodyHeader.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="mb-4">
+    <span class="category-header-large color-blue">
+      {{ fullTitleMessage }}
+    </span>
+    <div class="float-right">
+      <slot name="buttons"></slot>
+    </div>
+    <div>
+      {{ subTitle }}
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed } from "vue";
+export default {
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    subTitle: {
+      type: String,
+      required: false,
+    },
+    recordsCount: {
+      type: Number,
+      required: false,
+    },
+  },
+  setup(props: any) {
+    const fullTitleMessage = computed(() => {
+      if (props.recordsCount) {
+        return `${props.title} (${props.recordsCount})`;
```

Comment:
I see, `recordsCount` is not required (line 29), so if `recordsCount` is not passed then this will return `title (undefined)` right?

Reply:
also. for body headers like this 
![image](https://user-images.githubusercontent.com/77353155/153289634-adee0dcf-3f69-430c-9935-a92d8997c626.png)
there is no count, so we don't want the brackets themselves. right?